Sources/Datadog/RUM/Instrumentation/Views/SwiftUI/SwiftUIViewHandler.swift

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import UIKit
+
+/// Publisher generating RUM Commands on `SwiftUI.View` events.
+internal protocol SwiftUIViewHandler: RUMCommandPublisher {
+    /// Respond to a `SwiftUI.View.onAppear` event.
+    func onAppear(identity: String, name: String, path: String, attributes: [AttributeKey: AttributeValue])
+
+    /// Respond to a `SwiftUI.View.onDisappear` event.
+    func onDisappear(identity: String)
```

Comment:
nit: for consistency with other "handler" interfaces we could prefix these methods with `notify_` (or remove this prefix in other handlers): see `UIKit` views + actions handlers and `URLSession` tracing + RUM handlers.